src/zenml/integrations/mlflow/steps/mlflow_deployer.py

```diff
@@ -205,6 +220,132 @@ def mlflow_model_deployer_step(
     return new_service
 
 
+@step(enable_cache=False)
+def mlflow_model_registry_deployer_step(
+    params: MLFlowDeployerParameters,
+) -> MLFlowDeploymentService:
+    """Model deployer pipeline step for MLflow.
+
```

Comment:
Maybe you could add a few more details describing what this step is doing differently than the existing MLflow model deployer step ?